tbx/impact_reports/models.py

```diff
@@ -2,59 +2,66 @@
 from django.utils.text import slugify
 
 from tbx.core.utils.fields import StreamField
+from tbx.core.utils.models import ColourThemeMixin, SocialFields
 from tbx.impact_reports.blocks import ImpactReportStoryBlock
+from tbx.people.models import ContactMixin
 from wagtail.admin.panels import (
     FieldPanel,
     InlinePanel,
     MultiFieldPanel,
     TitleFieldPanel,
 )
-from wagtail.fields import RichTextField
 from wagtail.models import Page
 from wagtail.search import index
 
 
-class ImpactReportPage(Page):
+class ImpactReportPage(ColourThemeMixin, ContactMixin, SocialFields, Page):
     template = "patterns/pages/impact_reports/impact_report_page.html"
 
-    strapline = models.CharField(max_length=255)
-
     hero_image = models.ForeignKey(
         "images.CustomImage",
         null=True,
-        blank=True,
         on_delete=models.SET_NULL,
         related_name="+",
         help_text="This is for the illustration only. Use an image with dimensions of 571x700",
     )
 
-    introduction_title = models.CharField(max_length=255, default="Introduction")
-    introduction = RichTextField(blank=True)
+    hero_caption = models.CharField("caption", max_length=255, blank=True)
```

Comment:
added to be consistent with the design and work page image, which this is similar to.